modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/portlet/action/EditAssetCategoryFriendlyURLMVCActionCommand.java

```diff
@@ -126,6 +127,33 @@ protected void doProcessAction(
 
 				newUrlTitleMap.put(LocaleUtil.toLanguageId(locale), urlTitle);
 			}
+			else if ((urlTitle != null) && urlTitle.equals("")) {
```

Comment:
We should use `StringPool.BLANK`
```suggestion
			else if ((urlTitle != null) && urlTitle.equals(StringPool.BLANK)) {
```